### DISCUSSION TITLE
How to add new organism

### DISCUSSION THREAD

**Discussion by deandade** [wikipathways/wikipathways-help#120] [Q&A] [answered]
I'm learning how to create and submit pathways because I will most likely need it in the future. I am working with Solanum tuberosum (potato) and that organism is not available in the list. Can I add it and if so, how do I do it?

Best regards,
João Trindade

**Comment by khanspers** [comment, marked as answer]
Hi João. Thank you for your interest in WikiPathways. 

There are a few aspects of organism support in WikiPathways/PathVisio:
- In order to curate pathways for your own data visualization or presentation purposes, you don't necessarily need an identifier mapping database. If you can annotate each Gene Product node with identifiers that match your dataset exactly, you can then use the pathway models for visualization purposes in for example Cytoscape. You also have the option to add custom identifier mappings in PathVisio, stored as a text file. For example, you could use this to add mappings between gene and protein identifiers for Solanum tuberosum. This is available under Data > Identifier Mapping Setup > Add.
- Adding pathways to WikiPathways for an organism that is currently unsupported would first require [BridgeDb](https://www.bridgedb.org/) support for Solanum tuberosum. 

It would be great to know more about your project, and how you envision using WikiPathways.

**Reply by egonw** [reply]
I filed a feature request for BridgeDb, see https://github.com/bridgedb/BridgeDb/issues/244 and https://github.com/bridgedb/create-bridgedb-genedb-config/issues/4

**Reply by egonw** [reply]
@deandade, I also added it now to the WikiPathways Classic website: https://classic.wikipathways.org/index.php?title=Special%3ABrowsePathways&browse=Solanum_tuberosum&tag=---&view=thumbs (empty list at this moment)

Now, as @khanspers indicated, you can now create GPML files (with PathVisio) and you can start using Ensembl or NCBI Gene identifiers, but you will likely not be able to set the correct species in PathVisio. However, you can make this edit in a plain text or XML editor. I often set it to 'Homo sapiens' in PathVisio and then fix the species in a regular editor.

So, please let me know when you are ready to upload a first _Solanum tuberosum_ pathway to WikiPathways :)
